### DISCUSSION TITLE
How to query specific workflow in Argo Workflow UI

### DISCUSSION THREAD

**Discussion by pockyhe** [argoproj/argo-workflows#11059] [Q&A]
I would like to search a specific workflow in Argo Workflow UI, such as quick-start-ded34. However, I didn't find any filter in Argo Workflow UI
<img width="177" alt="image" src="https://github.com/argoproj/argo-workflows/assets/64390829/881f9e64-c656-4cc6-9a24-d5070822390e">
How can I query specific workflow in Argo Workflow UI? Thank you

**Comment by terrytangyuan** [comment]
Do you mean querying by name? If so, it's not available but you can just use your browser for find/search.
